src/engine/apply-changeset/index.ts

```diff
@@ -13,10 +14,11 @@ export const applyChangesetTask = (context: ApplyChangesetContext): Listr => {
         task: (ctx, task): Listr => {
           return task.newListr(
             [
-              createLoadChangesetTask(),
-              createRemoveEntitiesTask(),
-              createAddEntitiesTask(),
-              createChangeEntitiesTask(),
+              applyComputeIdsTask(),
+              applyLoadChangesetTask(),
```

Comment:
This will not throw an error, as it counts the changeset items before the changeset is loaded